### PR TITLE
Add home page css

### PR DESCRIPTION
**Pull request by jordan-sullivan**
## Summary:
Added CSS to all pages in application. 
Also added "Shake it up" button- will need to add a test on this in Cypress. 

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [ X ] New feature

## Where are the Changes:
All CSS files as well as a few re-names of class components. I will check to make sure all Cypress is running and adjust tests as needed tomorrow. 

## How was this tested:
UX/UI visual. 

## What are the relevant tickets:
Divide and conquer tickets for CSS mostly complete now.

### CODE REVIEW

File: src/components/AmarettoContainer/styles.css [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
+    font-family: 'Montserrat', sans-serif;
 }
 
 .amarettoButton:hover {
   cursor: pointer;
-  transform: scale(1.2);
-  transition: all 0.3s;
-  filter: brightness(120%);
+  transform: scale(1.1);
+  transition: all .8s;
```
Comment (kendallm360) [inline src/components/AmarettoContainer/styles.css:19]: Glad to see we added some styling to hovers. Small change but deadly

File: src/components/DetailCards/styles.css [hunk 2]
```diff
@@ -2,64 +2,48 @@
...
-    grid-area: 1 / 2 / 2 / 4;
-    font-size: 80px;;
+.stats {
+    margin-top: 3%;
+    margin-right:5%;
+    margin-left: 5%;
+    display:flex;
+    flex-direction: column;
+    background: #F6F6F6;
+    border: 20px solid white;
```
Comment (kendallm360) [inline src/components/DetailCards/styles.css:14]: This was a very slick idea to get the white box around each of the images

File: src/components/randomDrinkCards/RandomDrinkCards.css [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+   height: 40%;
+   width: 40%;
+   margin-top: 10%;
+}
+
+.drink-card:hover {
+   cursor: pointer;
+   transform: scale(1.1);
+   transition: all .8s;
+   box-shadow: 15px 15px 10px rgba(0, 0, 0, 0.3);
```
Comment (kendallm360) [inline src/components/randomDrinkCards/RandomDrinkCards.css:23]: A lot of these styles I have not really played with before so I am interested to see what it looks like

**Review by kendallm360 — APPROVED** [review]

**Review by nhodnett — APPROVED** [review]
Incredible work! Looking really good!

**Pull request merged**